src/main.cpp

```diff
@@ -478,33 +467,9 @@ void ledOff(PWMDriver * driver, LEDStrip * strip, uint8_t channelOffset)
   }  
 }
 
-void ledColour(PWMDriver * driver, LEDStrip * strip, uint8_t channelOffset)
-{
-  if (strip->channels == 1) 
-  {
-    driver->colour(strip->index, channelOffset, strip->colour[0]);
-  }
-  else if (strip->channels == 2) 
-  {
-    driver->colour(strip->index, channelOffset, strip->colour[0], strip->colour[1]);
-  }
-  else if (strip->channels == 3) 
-  {
-    driver->colour(strip->index, channelOffset, strip->colour[0], strip->colour[1], strip->colour[2]);
-  }
-  else if (strip->channels == 4) 
-  {
-    driver->colour(strip->index, channelOffset, strip->colour[0], strip->colour[1], strip->colour[2], strip->colour[3]);
-  }
-  else if (strip->channels == 5) 
-  {
-    driver->colour(strip->index, channelOffset, strip->colour[0], strip->colour[1], strip->colour[2], strip->colour[3], strip->colour[4]);
-  }  
-}
-
 void ledFade(PWMDriver * driver, LEDStrip * strip, uint8_t channelOffset)
 {
-  if ((micros() - strip->lastCrossfade) > g_fade_interval_us)
+  if ((micros() - strip->lastfadeUs) > strip->fadeIntervalUs)
```

Comment:
i had totally forgotten we had switched this to micros - i was writing the description based on millis... and explains the rather larger number i have for the fade on my smart bulb.

*micros is definitely the better one for this though.